src/wazuh_db/schema_agents.sql

```diff
@@ -366,7 +366,8 @@ CREATE TABLE IF NOT EXISTS sync_info (
     last_attempt INTEGER DEFAULT 0,
     last_completion INTEGER DEFAULT 0,
     n_attempts INTEGER DEFAULT 0,
-    n_completions INTEGER DEFAULT 0
+    n_completions INTEGER DEFAULT 0,
+    checksum TEXT NOT NULL CHECK (checksum <> '')
```

Comment:
This is something to discuss with the team:
Should this field have a more explicit name as "agent_checksum"? Also in the parameter names used in the methods that write the table.
It´s true that this is the checksum of the synchronization, so being inside of this table it could be enough with just "checksum". But it´s difficult to understand that this isn´t the checksum of any table at this specific moment.

Reply:
Maybe, we could use something like `last_global_checksum`

Reply:
Also, we should update the declaration to something like :
`... TEXT NOT NULL DEFAULT ' ' `. Or `... TEXT NOT NULL DEFAULT 'legacy' ` because the old agents will never use this value and if it can't be empty or NULL, the update query will fail.

Reply:
I renamed it as `last_global_checksum` and let the default value as `legacy`